### ISSUE TITLE
normalize ct/state

### ISSUE THREAD

**Issue by Vaccinebot** [CAVaccineInventory/vaccine-feed-ingest#518]
[![learn our pipeline: normalize](https://img.shields.io/static/v1?label=learn%20our%20pipeline&message=normalize&style=social)](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#normalize)

Blocked by #498

Normalize existing [`.ndjson`](http://ndjson.org/) records into [the Vaccinate The States data schema](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Normalized-Location-Schema).

Read all files in the directory passed as the second argument (`sys.argv[2]`), transform each record into the normalized schema (filling in as much detail as possible), then output them to new files in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the normalize stage and how to get set up for development!

### Tips

1. Fetch and parse data for this site before you start developing:
    ```sh
    poetry run vaccine-feed-ingest fetch ct/state
    poetry run vaccine-feed-ingest parse ct/state
    ```

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest normalize ct/state
    ```

### Example
[Parsed files for `ak/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/parsed) are converted to [normalized files](https://github.com/CAVaccineInventory/vaccine-feed-ingest-results/tree/main/ak/arcgis/normalized)

**Comment by rascalking** [comment]
Dibs

**Comment by rascalking** [comment]
Some of the sites have TTY phone numbers...should those be included?

**Comment by bryanculbertson** [comment]
@rascalking Ideally we should include TTY. However we aren't exposing that data on the website yet so it is lower priority.

I don't think there is a good way of indicated it is TTY in our phone field, so I think the best way will be to put it in the `other` field with a "TTY" at the end.

**Comment by rascalking** [comment]
@bryanculbertson After further reading through the parsed data, they have three sets of phone fields...`site_phones`, `agency_phones`, and `program_phones`.  `program_phones` is always empty, and it looks like `agency_phones` is the only one with the tty numbers.  The agency seems to be the health system running the site, not the site proper, but I'm assuming if the agency has only one tty phone, it's okay to use it for everything.

I'm gonna go with including the agency tty phone in the contacts, but not the rest of the agency phone numbers.  Let me know if you want to go a different way.

**Comment by rascalking** [comment]
More questions...there's a `description` field that usually contains html.  Safe to assume we should convert it to plain text as part of normalization?

**Comment by bryanculbertson** [comment]
@rascalking Yes, please strip the HTML tags and just include the text. Thank you!
